src/ecrc-frontend/src/components/page/informationreview/InformationReview.stories.js

```diff
@@ -61,43 +70,102 @@ const newPayload = {
+  const mock = new MockAdapter(axios);
+  const API_REQUEST = "/ecrc/private/checkShare?requestGuid=unique123";
+
+  mock.onGet(API_REQUEST).reply(404, { message: "No share available." });
+
+  sessionStorage.setItem("validator", "secret");
+  sessionStorage.setItem("uuid", "unique123");
+
+  return props.children({ page });
+};
+
+const LoadData = props => {
+  const mock = new MockAdapter(axios);
+  const API_REQUEST = "/ecrc/private/checkShare?requestGuid=unique123";
+
+  mock.onGet(API_REQUEST).reply(200, {
+    oldOrg: "Old Org",
+    oldCRCExpiration: "2021-10-22"
+  });
+
+  sessionStorage.setItem("validator", "secret");
+  sessionStorage.setItem("uuid", "unique123");
+
+  return props.children({ page });
+};
 
 storiesOf("Information Review", module)
+  .add("Display Share Option", () => (
+    <LoadData props={page}>
+      {data => (
+        <MemoryRouter>
+          <InformationReview page={data.page} />
+        </MemoryRouter>
+      )}
+    </LoadData>
+  ))
   .add("NonSchedule D Default", () => (
-    <MemoryRouter>
-      <InformationReview page={page} />
-    </MemoryRouter>
+    <FailData props={page}>
+      {data => (
+        <MemoryRouter>
+          <InformationReview page={data.page} />
+        </MemoryRouter>
+      )}
+    </FailData>
   ))
   .add("Schedule D Default", () => (
-    <MemoryRouter>
-      <InformationReview
-        page={{
-          ...page,
-          applicant: {
-            ...applicant,
-            organizationFacility: "PBS WIPB"
-          }
-        }}
-      />
-    </MemoryRouter>
+    <FailData props={page}>
+      {data => (
+        <MemoryRouter>
+          <InformationReview
+            page={{
+              ...data.page,
+              applicant: {
+                ...applicant,
+                organizationFacility: "PBS WIPB"
+              }
+            }}
+          />
+        </MemoryRouter>
+      )}
+    </FailData>
   ))
   .addParameters({ viewport: { defaultViewport: "mobile2" } })
   .add("NonSchedule D Mobile", () => (
-    <MemoryRouter>
-      <InformationReview page={page} />
-    </MemoryRouter>
+    <FailData props={page}>
+      {data => (
+        <MemoryRouter>
+          <InformationReview page={data.page} />
+        </MemoryRouter>
+      )}
+    </FailData>
   ))
   .add("Schedule D Mobile", () => (
-    <MemoryRouter>
-      <InformationReview
-        page={{
-          ...page,
-          applicant: {
-            ...applicant,
-            organizationFacility: "PBS WIPB"
-          }
-        }}
-      />
-    </MemoryRouter>
+    <FailData props={page}>
+      {data => (
+        <MemoryRouter>
+          <InformationReview
+            page={{
+              ...data.page,
+              applicant: {
+                ...applicant,
+                organizationFacility: "PBS WIPB"
+              }
+            }}
+          />
+        </MemoryRouter>
+      )}
+    </FailData>
+  ))
+  .add("Display Share Mobile", () => (
```

Comment:
should we have a faildata version for mobile?

Reply:
FailData is wrapped around mobile. All FailData is simulating a failed check for share, so the component does not display (which is the default)